src/streams/logs.ts

```diff
@@ -0,0 +1,69 @@
+import { Readable } from 'stream';
+import { listOnePageLogResources } from '../api/logs';
+import { LogApiResource, Sid, GotClient } from '../types';
+import { LogsConfig } from '../types/logs';
+
+export class LogsStream extends Readable {
+  _buffer: Array<LogApiResource>;
+  _interval: NodeJS.Timeout | undefined;
+  pollingFrequency = 1000;
```

Comment:
should this be a `private` property with an explicit `setter` instead? That way you can reset the interval. Right now if I would change the `pollingFrequency` it would actually not impact anything